deps/libgit2.mk

```diff
@@ -80,7 +80,7 @@ endif
 
 clean-libgit2:
 	-$(MAKE) -C $(BUILDDIR)/$(LIBGIT2_SRC_DIR) clean
-	-rm -f $(LIBGIT2_OBJ_TARGET)
+	-rm -f $(build_shlibdir)/libgit2* $(build_shlibdir)/libssl* $(build_shlibdir)/libcrypt
```

Comment:
`make clean` should not leave things in a state that lead to an error when you do `make install` again, which is what happens without this.